doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,31 @@
+= Transforming coordinate systems
+
+Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+
+//TODO: Should we add one or more sentences about the mathematical operations involved?
+
+**Corresponding messages**
+
+``GroundTruth::moving_object::base::position``::
+Defines the position of the vehicle’s reference point in world coordinates.
+In Open Simulation Interface, the position of an object is defined by the coordinates of the center of the object's bounding box.
```

Comment:
"Object coordinate system:
Coordinate system for local object coordinates. 
The origin of the corresponding coordinate systems is not necessarily identical to the center of the object's bounding box.
For example with vehicles, the origin is the middle of the rear axle projected onto the ground."
@pmai we should discuss this in our next CCB meeting.

Reply:
I am not sure what change you are suggesting. Sounds like a quote from the coordinate-system topic.

Reply:
@pasched to be done

Reply:
@pasched to be done.

Reply:
OSI CCB: 
- review in CCB: 
```suggestion
In Open Simulation Interface, the position of an object is defined by the coordinates of the center of the object's 3D bounding box.
```

Reply:
@pasched Todo